src/modules/home/components/property/property-card.tsx

```diff
@@ -13,17 +15,48 @@ import 'swiper/css/pagination';
 import './swiper.css';
 
 import { EffectFade, Navigation, Pagination } from 'swiper/modules';
+import { axiosInstance } from '#/configs';
+import { useWishlistedProperties } from '../../api/get-wishlisted-properties';
 
-export function ApartmentCard(apartment: Accommodation) {
-  const handleFavoriteClick = (event: React.MouseEvent) => {
+export function ApartmentCard({ id, name, location, pricePerNight, images }: Accommodation) {
+  const { data } = useWishlistedProperties();
+  const navidate = useNavigate();
+  const [isLiked, setIsLiked] = useState(false);
+
+  useEffect(() => {
+    if (data?.data) {
+      const likedFromData = data.data.find((item) => item.id === id);
+      setIsLiked(!!likedFromData);
+    }
+  }, [data, id]);
+
+  const handleFavoriteClick = async (event: React.MouseEvent) => {
     event.stopPropagation();
     event.preventDefault();
+    const isUserExist = localStorage.getItem(`isUserLoggedIn`);
```

Comment:
Checking the user login state should be done by checking if the access token does exist in cookies or not. Because there might be a case when there is no token but `isUserLoggedIn` is still true.

Reply:
I suppose you'll create a new ticket and PR to update the login state check?